packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -2,110 +2,120 @@ import React from 'react';
 import { withTheme } from '../../themes';
 import { Themeable } from '../../types';
 import { CodeEditorProps } from './types';
-import { registerSuggestions } from './suggestions';
-import ReactMonaco from 'react-monaco-editor';
-import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+// import { registerSuggestions } from './suggestions';
+// import ReactMonaco from 'react-monaco-editor';
+// import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+import Editor, { loader } from '@monaco-editor/react';
+
+loader.config({
+  paths: {
+    // TODO: use public path, or something? for when grafana is hosted under a subpath
```

Comment:
and check the CDN settings ;)

Reply:
You mean if Grafana is serving static assets from a CDN path? I presume I can use this? 
https://github.com/grafana/grafana/blob/master/public/app/index.ts#L6-L8